scss/button/_layout.scss

```diff
@@ -118,9 +118,21 @@
     }
 
     //DropDown Button
-    kendo-dropdownbutton.k-state-focused,
-    kendo-dropdownbutton:focus {
-        outline: 0;
+    .k-dropdownbutton {
```

Comment:
Seems to be the same question that we had for the `button-group` and `buttongroup`, no? telerik/kendo-angular-buttons#45